src/main/scala/io/delta/DeltaExtensions.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2019 Databricks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.delta
+
+import org.apache.spark.sql.SparkSessionExtensions
+import org.apache.spark.sql.delta.analysis.DeltaAnalysis
+
+class DeltaExtensions extends Function1[SparkSessionExtensions, Unit] {
```

Comment:
Can you add docs to this? @zsxwing will use this to inject SQL parser commands as well